hack/kind_e2e.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+KIND_LOG_LEVEL="1"
+
+if [ -n "${DEBUG}" ]; then
+  set -x
+  KIND_LOG_LEVEL="6"
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OPENELB_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+source "${OPENELB_ROOT}"/hack/lib/init.sh
+
+cleanup() {
+  kind delete cluster \
+    --verbosity="${KIND_LOG_LEVEL}" \
+    --name "${KIND_CLUSTER_NAME}"
+}
+
+trap cleanup EXIT
+
+export KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-openelb-e2e}
+
+if ! command -v kind --version &> /dev/null; then
+  echo "kind is not installed. Use the package manager or visit the official site https://kind.sigs.k8s.io/"
+  exit 1
+fi
+
+echo "Creating Kubernetes cluster with kind"
+
+export K8S_VERSION=${K8S_VERSION:-v1.24.7}
```

Comment:
We can set this as a version matrix in GitHub workflows.

```
  e2e:
...
    strategy:
      matrix:
        # Here support the latest three minor releases of Kubernetes, this can be considered to be roughly
        # the same as the End of Life of the Kubernetes release: https://kubernetes.io/releases/
        # Please remember to update the CI Schedule Workflow when we add a new version.
        k8s: [ v1.25.0, v1.26.0, v1.27.3 ]


....


      - name: setup e2e test environment
        run: |
          export CLUSTER_VERSION=kindest/node:${{ matrix.k8s }}
          hack/kind_e2e.sh

```

Reply:
ok, I will integrate it into the github workflow in the next PR. If the k8s version is not specified, we still use the default 1.24.7.